src/Components/Game/Game.js

```diff
@@ -3,11 +3,11 @@ import './Game.css'
 import '../../assets/whos-that-pokemon_.mp3'
 import {getSinglePokemon} from '../../ApiCalls'
 
-const Game = ({pokemons, get4RdmPokemon}) => {
+const Game = ({ get4RdmPokemon}) => {
   const [singlePokemon, setSinglePokemon] = useState({});
   const [winCounter, setWinCounter] = useState(0);
-  const [pokemonChoices, setPokemonChoices] = useState(pokemons)
-
+  const pokemons = get4RdmPokemon();
+  const [pokemonChoices, setPokemonChoices] = useState(pokemons);
```

Comment:
I like this refactoring to a single parameter, my vote is to keep it. Single concern would be some conflicts/failing test from Corbin's branch in the future.